Dockerfile

```diff
@@ -16,6 +16,8 @@ RUN pip install pip --upgrade
 # Install project packages
 RUN pip install -r requirements.txt
 RUN pip install -r test_requirements.txt
+RUN pip3 install -r requirements.txt
+RUN pip3 install -r test_requirements.txt
```

Comment:
Are the python2 and python3 dependencies being roped in to allow for running the test suite against both at the same time?